src/cljs/puppetlabs/metrics/dashboard/metrics_box.cljs

```diff
@@ -115,7 +125,9 @@
 
     (.. box-node
         (select ".countertext")
-        (html (format (last datavals))))
+        (html (if-let [last-val (last datavals)]
+                (format-fn last-val)
+                "??")))
```

Comment:
Before calling `format` on `(last datavals)` when `datavals` was empty would cause a `NaN` error, this essentially changes the "default" value shown for a metric in the metric box from NaN -> `??` which is how the PDB dashboard formats that case today